index.html

```diff
@@ -20,140 +20,21 @@
-                          engineering skills acquired during my Software Engineering
-                          Immersive course in General Assembly bring these solutions to
-                          life.
-                        </p>
-                        <h2>Skills</h2>
-                        <p>
-                        Javascript, HTML5, CSS, Node.js, React, express, ejs, mongodb,
-                        mongoose, jQuery, Git, Github
-                        </p>
-                  </div>
-                  <div class="tab-pane" id="tab2">
-                      <h2>Projects</h2>
-                      <a
-                        href="https://github.com/Hantasmagoria/Hantasmagoria.github.io/blob/master/documentations/overdoze.md"
-                        title="Project 1 for Software Engineering Immersive @ General Assembly.
-                              Platform game developed using HTML5 and Javascript.
-                              Assets used are from opengameart.org"
-                      >Overdoze</a>
-                      <a
-                        href="https://overdozews.herokuapp.com/"
-                        title="Project 2 for Software Engineering Immersive @ General Assembly.
-                              Database-enabled website for leaderboard and score tracking.
-                              Inspired by design at osu.ppy.sh"
-                      >ODDB</a>
-                      <a
-                        href="https://github.com/Hantasmagoria/UnrehearsedAntihistoricalValedictorian"
-                        title="Discord bot catered for the needs of the Invictus discord server. 
-                              Uses the discord.js library, hosted on heroku."
-                        >Shana the Discord Bot</a>
-                        <!-- 
-                <a href="#">Project 4</a>
-                <a href="#">Project 5</a>
-                <a href="#">Project 6</a>
-                <a href="#">Project 7</a>
-                <a href="#">Project 8</a>
-                <a href="#">Project 9</a>
-                <a href="#">Project 10</a>
-                <a href="#">Project 11</a> 
-                        -->
-                      <a
-                        href="https://github.com/Hantasmagoria/mcojn"
-                        title="A Javascript Bookmarklet for restructuring the data in the DOM within the facility booking system in MCOnline.sg"
-                        >MCOnline JN script</a>
-                  </div>
-                  <div class="tab-pane" id="tab2">
-                      <h2>Contact</h2>
-                      <p>
-                        <a href="mailto:farhan.daud.hyperfusion@gmail.com"
-                          >farhan.daud.hyperfusion@gmail.com</a>
-                      </p>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
+        <div class="row p-2" id="root">
+          <!-- React Components here -->
```

Comment:
as above; this comment will remind humanity once more, my React attaches to #root.